services/core-api/tests/status_code_gen.py

```diff
@@ -39,6 +40,11 @@ def RandomMineRegionCode():
     return random.choice([x.mine_region_code for x in db.session.query(MineRegionCode).all()])
 
 
+def RandomEMLIContactTypeCode():
+    return random.choice(
+        [x.emli_contact_type_code for x in db.session.query(EMLIContactType).all()])
```

Comment:
Something we need to do on a lot of our factories but with this one can you filter it by active=True. this has caused issues in the past in some tests were codes have been marked as inactive and the code changed around that causing the test to fail.

Reply:
Added, to ensure the record is an active one.